duckduckgo/privacy-dashboard
# Site may be malicious - copy change

Asana: https://app.asana.com/0/38424471409662/1208863401278206/f
**Reviewer:**

## Description:

- Copy change on malicious site warning: “Site May Be Deceptive” => “Site May Be Malicious"

<img width="725" alt="image" src="https://github.com/user-attachments/assets/b6d6f737-5d3d-4672-b11d-13d0bddb71c7">

## Steps to test this PR:

1. Visit the preview or run debugger locally with `npm run build && npm run preview`
2. Open the phishing or malware screen
3. Confirm that first line on tracker breakdown reads “Site May Be Malicious"
